samples/nrf9160/modem_shell/src/cloud/cloud_lwm2m.c

```diff
@@ -197,6 +200,24 @@ static int cloud_lwm2m_init(const struct device *unused)
 	return 0;
 }
 
+static void cloud_lwm2m_rd_client_update_lifetime(int srv_obj_inst)
+{
+	char pathstr[MAX_RESOURCE_LEN];
+	uint32_t current_lifetime = 0;
+	uint32_t lifetime = CONFIG_LWM2M_ENGINE_DEFAULT_LIFETIME;
```

Comment:
Do we need this variable? I started to look where it's modified and seems it's not.

Reply:
No, it's not necessarily needed, `CONFIG_LWM2M_ENGINE_DEFAULT_LIFETIME` could be used directly instead. It's a matter of taste which one is better, but the generated code is identical.

Reply:
Ok, compiler optimizes the variable out. I guess you should make it `const` if you want to keep it so reader doesn't need to check whether it's modified.